source/openapi-admin-v3.yaml

```diff
@@ -5632,6 +5771,157 @@ components:
+          type: string
+          description: |-
+            The human-readable name supplied when 
+            [creating the Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-an-edge-server-instance) 
+            instance.
+        user_id:
+          type: string
+          description: |-
+            The Edge Server itself is a user. This is the unique identifier of 
+            the Edge Server user, of type ObjectId, as a string.
+        status:
+          type: string
+          enum:
+            - "active"
+            - "deleted"
+            - "failed"
+            - "registered"
+            - "unregistered"
+          description: |-
+            The current status of the Edge Server instance.
+        hostname:
+          type: string
+          description: The public-addressable DNS of the Edge Server instance.
+        last_connection:
+          type: string
+          description: |-
+            A string representing the timestamp at which the Edge Server 
+            instance last connected to Atlas.
+        query: 
+          type: object
+          description: |-
+            The current [Sync queries](https://www.mongodb.com/docs/atlas/app-services/edge-server/configure/#specify-the-edge-server-s-query) 
+            for this Edge Server instance. This object maps collection 
+            names to the query for that collection.
+        package_version:
+          type: string
+          description: |-
+            The version of the Edge Server code that the instance is
+            currently running. If this is more than one major version
+            behind the latest version, the Edge Server instance enters
+            a "failed" state.
+        latest_package_version:
+          type: string
+          description: |-
+            The latest version of the Edge Server code that is available.
+        metrics:
+          $ref: "#/components/schemas/EdgeServerMetrics"
+      example:
+        id: 6622b8f5118368b943d228a9
+        created: 2024-04-19T08:51:00Z
+        name: Edge-Store-42
+        user_id: 6622b8f5118368b943d2288a
+        status: active
+        hostname: ec2-13-58-70-88.us-east-2.compute.amazonaws.com
+        last_connection: 2024-04-19T09:51:00Z
+        query: { "Item": '(owner_id == "641cb3d99e425c794ef69315")' }
+        package_version: v0.21.2
+        latest_package_version: v0.22.2
+        metrics: {}
+    EdgeServerMetrics:
+      type: object
+      description: |-
+        The metrics for this Edge Server instance. [PLACEHOLDER: Provide more
+        details about metrics when available.]
+    EdgeServerCreateResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: 6622b8f5118368b943d228a9
+          description: |-
+            The unique identifier of the Edge Server instance, of type 
+            ObjectId, as a string. You can use this identifier when configuring
```

Comment:
actually, you wouldn't use the edge server ID to configure permissions; you'd use it to delete/ update the edge server. you'duse the user id to configure perms